docs/source/customization.rst

```diff
@@ -0,0 +1,52 @@
+Tree handler customization
+==========================
+
+What to do if a time comes and you need some fancy stuff done to tree items that
+*django-sitetree* does not support?
+
+.. _tree-custom:
+
+It might be that you need some special tree items ordering in a menu, or you want to render
+in a huge site tree with all articles titles that are described by one tree item in Django admin,
+or god knows what else.
+
+*django-sitetree* can facilitate on that as it allows tree handler customization
+with the help of `SITETREE_CLS` setting.
+
+1. Subclass `sitetreeapp.SiteTree` and place that class into a separate module for convenience.
+2. Override methods you need for customization (usually `.apply_hook()`).
```

Comment:
From that description I don't really understand how .apply_hook() comes into play.

Reply:
`.appy_hook()` is most generic — this description doesn't necessarily covers your very case from #263 
Rather you override `.get_permissions()`.

Reply:
Yeah, thats right. I just wanted to point out, that somebody who reads this sentence might not understand if he has to overwrite .apply_hook() or if this is just one option to deal with a usecase.

But I guess if people are doing such things they must have a look at the code and then everything is pretty self-explanatory anyways.

Reply:
I see. If you can propose a better wording, I'd be glad.

Reply:
I'd be a little bit more explicit, like this:

You can now overwrite `.apply_hook()` to manipulate values at pre-defined hooks. You can also overwrite any other methods to customize to your exact needs.

Reply:
Done. Thank you.